src/ec/packages/ec-component-accordion2/accordion2.story.js

```diff
@@ -1,36 +1,49 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { buttonLabels, getExtraKnobs } from '@ecl-twig/story-utils';
```

Comment:
We try to separate the import of objects of functions from the one about specs and resources, just by an empty line.

Reply:
can you please do just the same that is done in the deprecated accordion, regarding the organisation of this import statements? It's a minor thing, eh, but maybe exactly because of this we shouldn't let it go.

Reply:
oke i have re-ordered them in this import statement

Reply:
@Joosthe please check this remarks https://github.com/ec-europa/ecl-twig/pull/381#discussion_r396542121

Reply:
I did this exactly like the deprecated accordion or what do you mean by this ?

Reply:
ok indeed seems good to me, https://github.com/ec-europa/ecl-twig/pull/381#discussion_r396542121 was not clear to me.